src/app/features/login/login-dialog.component.ts

```diff
@@ -36,10 +42,40 @@ export class LoginDialog implements ModalComponent<LoginDialogParameters> {
 		if (document.body.classList.contains('modal-open')) {
 			document.body.classList.remove('modal-open');
 		}
-		this.dialog.close({
-			username: this.parameters.username,
-			password: this.parameters.password,
-			server:   this.parameters.server
+
+		this.checkConnection().subscribe(() => {
+			this.dialog.close({
+				isLogged: true,
```

Comment:
Set isLogged to false when there is an issue

Reply:
ok, done